sympy/geometry/line.py

```diff
@@ -2364,28 +2369,44 @@ def equation(self, x='x', y='y', z='z', k='k'):
         y : str, optional
             The name to use for the y-axis, default value is 'y'.
         z : str, optional
-            The name to use for the x-axis, default value is 'z'.
+            The name to use for the z-axis, default value is 'z'.
 
         Returns
         =======
 
-        equation : tuple
+        equation : Tuple of simultaneous equations
 
         Examples
         ========
 
-        >>> from sympy import Point3D, Line3D
+        >>> from sympy import Point3D, Line3D, solve
+        >>> from sympy.abc import x, y, z
         >>> p1, p2 = Point3D(1, 0, 0), Point3D(5, 3, 0)
         >>> l1 = Line3D(p1, p2)
-        >>> l1.equation()
-        (x/4 - 1/4, y/3, zoo*z, k)
+        >>> eq = l1.equation(x, y, z); eq
+        (-3*x + 4*y + 3, z)
+        >>> solve(eq.subs(z, 0), (x, y, z))
+        {x: 4*y/3 + 1}
 
         """
-        x, y, z, k = [_symbol(i, real=True) for i in (x, y, z, k)]
+        if k is not None:
+            SymPyDeprecationWarning(
+                            feature="equation() no longer needs 'k'",
+                            issue=13695).warn()
```

Comment:
Arguments of SymPyDeprecationWarning() are they okay ?

Reply:
You will need to use `deprecated_since_version=1.2` argument as well.